ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/ViewExt.kt

```diff
@@ -1,4 +1,4 @@
-package com.asfoundation.wallet.ui.common
+package com.appcoins.wallet.ui.widgets
 
 import android.view.View
```

Comment:
This file probably could be in the :ui:common inside the extensions package, or even included in the file in there